### DISCUSSION TITLE
how to download large-v2 model ?

### DISCUSSION THREAD

**Discussion by zhuiyi6** [openai/whisper#1450] [Q&A]
UserWarning: C:\Users\Administrator.DESKTOP-DHKFNAB\.cache\whisper\small.pt exists, but the SHA256 checksum does not match; re-downloading the file

**Comment by zhuiyi6** [comment]
download:
https://github.com/openai/whisper/blob/f296bcd3fac41525f1c5ab467062776f8e13e4d0/whisper/__init__.py

i download large-v2 model,but i can't use this model.
there are so many small files  in this large-v2.zip,how to convert to xx.pt type to use it

**Comment by ThioJoe** [comment]
Pretty sure it downloaded automatically for me when running the script the first time
